docs/conf.py

```diff
@@ -10,9 +10,10 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("../"))
```

Comment:
It seems we can leave this commented and the docs still work. Can you double check this and, if so, comment these lines back again?

Reply:
SphinX couldn't find the pbparam files when comment them out. Therefore, I'm leaving as it is as agreed.